tf_files/aws/variables.tf

```diff
@@ -24,13 +24,13 @@ variable "base_ami" {
 variable "db_size"{
     default = 10
 }
-variable "db_password_userapi" {
+variable "db_password_fence" {
 }
 variable "db_password_gdcapi" {
 }
 variable "db_password_indexd" {
 }
-variable "userapi_snapshot" {
+variable "fence_snapshot" {
```

Comment:
Give the fence_snapshot a default value of "" - the snapshot only comes into play when we migrate the database from an old VPC to a new VPC.